app/views/docs/realtime.phtml

```diff
@@ -0,0 +1,341 @@
+<p>A list of all channels available you can subscribe to:</p>
+<table class="full text-size-small">
+    <thead>
+        <tr>
+            <td style="width: 180px">Channel</td>
+            <td>Description</td>
+        </tr>
+    </thead>
+    <tbody>
+        <?php foreach ($channels as $channel => $description) : ?>
+            <tr>
+                <td><?php echo $this->escape($channel); ?></td>
+                <td><?php echo $this->escape($description); ?></td>
+            </tr>
+        <?php endforeach; ?>
+    </tbody>
+</table>
+
+<h2><a href="/docs/realtime#custom-endpoint" id="custom-endpoint">Custom Endpoint</a></h2>
+<p>By default, the SDK will guess the endpoint of the Realtime endpoint when setting the endpoint of your Appwrite instance. If you are running a custom proxy and changed the endpoint of the WebSOcket server to something different than <b>wss://[ENDPOINT]/v1/realtime</b> you can call <b>setEndpointRealtime</b> on the Client SDK.</p>
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide margin-bottom" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>let sdk = new Appwrite();
+
+sdk.setEndpointRealtime('[ENDPOINT]');</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+
+        <div class="ide margin-bottom" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>final client = Client();
+client.setEndpointRealtime('[ENDPOINT]');</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-bottom" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client()
+client.setEndpointRealtime('[ENDPOINT]')</code></pre>
+        </div>
+    </li>
+</ul>
+<h2><a href="/docs/realtime#known-limitations" id="known-limitations">Known limitations</a></h2>
+<h3>Be careful with multiple quick changes of different subscribed channels.</h3>
+<p>The SDK creates a single WebSocket connection for all subscribed channels. Each time a channel is added or unsubscribed - the SDK currently creates a completely new connection and terminates the old one. Therefore, subscriptions to channels should always be done in conjunction with state management so as not to be unnecessarily built up several times by multiple components' life cycles.</p>
+
+<h3>Nested Documents.</h3>
+<p>Nested Documents will not trigger events related to their parent documents. Meaning, updates to a child document - will not trigger an event for the parent document. But it will trigger an event for the child document itself.</p>
+
+<h3>Server-Side?</h3>
+<p>We currently are not offering access to realtime with Server SDKs and an API key.</p>
```

Comment:
Can we allow the use with a JWT key?

Reply:
Yes, that will work out of the box 👍🏻

Reply:
So we should probably mention that on the tutorial? at least say that only an api-key is not yet supported, or have a complete section for server side integration? maybe just an external tutorial?